src/interface.ts

```diff
@@ -15,33 +15,65 @@ enum ResolveError {
   ResolverTimeout = "ResolverTimeout"
 }
 
-interface Intent {
-  intent: IntentName;
-  context: Context;
+/**
+ * the result of calling desktopAgent.resolve() is `Promise<ResolveResults>`
+ * the ResolveResult contains the list of apps which support the intent
+ */
+export interface RaiseIntentResults {
```

Comment:
Again, I don't think raising the intent returns data, especially not `any` data.

An intent for me is a capability you are invoking, with a success or failure indication. It's beauty, and usefulness, is in the simplicity to compose if it is fire-and-forget.

I thought we said that we are going to wait on the effort to define a more wide-ranging set of interop APIs that cover the RPC case?

Reply:
This was added as a suggestion (wont name names) but I am OK with leaving it off and handling any RPC-type stuff in another stream.

There is always a possibility of APP1 resolving an intent to APP2 and then APP2 raising a "response intent" that only APP1 could handle.